src/directwrite/mod.rs

```diff
@@ -238,12 +231,30 @@ impl crate::Rasterize for DirectWriteRasterizer {
     fn get_glyph(&mut self, glyph: GlyphKey) -> Result<RasterizedGlyph, Error> {
         let loaded_font = self.get_loaded_font(glyph.font_key)?;
 
-        match self.rasterize_glyph(&loaded_font.face, glyph.size, glyph.c) {
-            Err(err @ Error::MissingGlyph(_)) => {
-                let fallback_font = self.get_fallback_font(&loaded_font, glyph.c).ok_or(err)?;
-                self.rasterize_glyph(&fallback_font.create_font_face(), glyph.size, glyph.c)
-            },
-            result => result,
+        let fallback_font;
+        let (font, glyph_index) = iter::once(loaded_font)
+            .chain({
+                fallback_font =
+                    self.get_fallback_font(&loaded_font, glyph.character).map(Font::from);
+                &fallback_font
+            })
+            .find_map(|font| {
+                let glyph_index = self.get_glyph_index(&font.face, glyph.character);
+                glyph_index.map(|glyph_index| (font, Some(glyph_index)))
+            })
+            .unwrap_or((loaded_font, None));
+
+        // DirectWrite uses 0 for missing glyph symbols.
+        let is_missing_glyph = glyph_index.is_none();
```

Comment:
Same thing about just checking for `== 0`. If we can `.unwrap_or(0)`, we should also be able to check for `0`.

Can just create a constant too if we want to make it more clear.